.azure-pipelines.yml

```diff
@@ -0,0 +1,22 @@
+jobs:
+  - job: Windows
+    strategy:
+      matrix:
+        MSVC_2015:
+          MUMBLE_QT: qt5
+          MUMBLE_HOST: x86_64-pc-windows-msvc
+        MSVC_2015_NO_PCH:
+          MUMBLE_QT: qt5
+          MUMBLE_HOST: x86_64-pc-windows-msvc
+          MUMBLE_NO_PCH: 1
+    steps:
+    - script: git submodule --quiet update --init --recursive
+      displayName: 'Fetch submodules'
```

Comment:
Maybe put the display name first in all of these? Should be easier to identify/read them then.